src/plugins/intel_gpu/src/graph/impls/ocl/crop.cpp

```diff
@@ -41,16 +41,19 @@ struct crop_impl : typed_primitive_impl_ocl<crop> {
 
 public:
     static primitive_impl* create(const crop_node& arg, const kernel_impl_params& impl_param) {
-        const auto& primitive = arg.get_primitive();
-
         auto ew_params = get_default_params<kernel_selector::eltwise_params>(impl_param, 1);
         auto ew_optional_params = get_default_optional_params<kernel_selector::eltwise_optional_params>(arg.get_program());
 
         ew_params.operations.push_back(
             {{kernel_selector::eltwise_params::InputType::Buffer(0)}, kernel_selector::eltwise_mode::ASSIGN});
 
         const auto& input_layout = impl_param.input_layouts[0];
-        ew_params.inputs[0] = convert_data_tensor(input_layout, 1, primitive->offsets);
+        auto desc = impl_param.typed_desc<crop>();
+        if (arg.get_dependencies().size() > 1) {
```

Comment:
Here, we shouldnt use arg, but impl_param. So please change this to impl_param.input_layouts

Reply:
Done